controllers/ingressnodefirewallconfig_controller.go

```diff
@@ -86,9 +90,30 @@ func (r *IngressNodeFirewallConfigReconciler) Reconcile(ctx context.Context, req
 	}
 
 	if err := r.syncIngressNodeFwConfigResources(ctx, instance); err != nil {
```

Comment:
err = r.syncIngressNodeFwConfigResources

I'm pretty sure that otherwise, that error here is defined in a new scope and thus it will never make it to the return below.